### ISSUE TITLE
Reproduce issues

### ISSUE THREAD

**Issue by JonnesLin** [divyam3897/UCL#3]
Hi authors,

Ideally, we can get the same result as the paper reported if I set the cl_model to "mixup", right? But I only got 70.95% acc on this way :\

Could you provide more details?

**Comment by divyam3897** [comment]
Hi Jonnes,

Thank you for expressing interest in our work. Would you mind sharing further details with us? Specifically, we would like to know which dataset you are using and whether you have made any adjustments to the hyper-parameters, particularly alpha, in line with the settings outlined in the supplementary material of our paper.

**Comment by JonnesLin** [comment]
Hey Divyam,

Thank you for your response! I cloned the code and trained it with the TinyImageNet dataset. The only place I changed is the conv layer: I changed it from "self.conv1 = nn.Conv2d(3, self.inplanes, kernel_size=3, stride=1, padding=1, bias=False)" to "self.conv1 = nn.Conv2d(3, self.inplanes, kernel_size=7, stride=2, padding=3, bias=False)" as the comment implied.

**Comment by divyam3897** [comment]
Hi Jonnes,

After cloning the code, I performed a single test run with SimSiam and achieved a result of 77.41%, which is consistent with the reported results' margin of error. I retrieved this number from the log files and cross-verified it with a separate evaluation conducted via ```linear_eval_alltasks.py```.

**Comment by JonnesLin** [comment]
Hi Divyam,

I got a similar result when the self.conv1 is the following parameters: "kernel_size=3, stride=1, padding=1". But the comment shows it is used for the CIFAR datasets. So we still need to use the same parameters for the TinyImageNet? 

I appreciate your help!

Best,
Jones

**Comment by divyam3897** [comment]
Hi Jonnes,

In my test runs, both settings show similar performances.

**Comment by JonnesLin** [comment]
Ok! Thank you very much!
